doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3005,7 +3005,7 @@ the conversion in hypotheses :n:`{+ @ident}`.
    flags are either ``beta``, ``delta``, ``match``, ``fix``, ``cofix``,
    ``iota`` or ``zeta``. The ``iota`` flag is a shorthand for ``match``, ``fix``
    and ``cofix``. The ``delta`` flag itself can be refined into
-   :n:`delta {+ @qualid}` or :n:`delta -{+ @qualid}`, restricting in the first
+   :n:`delta [ {+ @qualid} ]` or :n:`delta - [ {+ @qualid} ]`, restricting in the first
```

Comment:
`@qualid` is not the only option here.

```
smart_global: [
| reference    (* referred to as qualid *)
| by_notation
]

by_notation: [
| ne_string IDENT_opt
]

ne_string: [
| STRING
]

IDENT_opt: [
| "%" IDENT
| empty
]
```

(lost my earlier, more detailed explanation :-( )

Reply:
Indeed, this corresponds to my remark here: https://github.com/coq/coq/pull/10763#issuecomment-532692533